tests/general/test_jnp_vs_np.py

```diff
@@ -0,0 +1,235 @@
+import pytest
```

Comment:
It would be good to have a brief description of this test since it doesn't fit in with the others. In some sense, it's a test of Jax, correct?

Reply:
This is correct. I'll add a comment. It's just an installment to check the functionality we need for correspondence up to a certain precision with numpy

Reply:
Yes we need this as a canary of sorts.